infra/aws/terraform/prow-build-cluster/resources/boskos/config.yaml

```diff
@@ -0,0 +1,26 @@
+# Copyright 2023 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  creationTimestamp: null
+  name: resources
+  namespace: test-pods
+data:
+  config: |
+    resources:
+    - names:
+        - k8s-aws-boskos-01
+      state: free
```

Comment:
I think that this should be dirty. cc @dims

Reply:
i think so as well! https://cs.k8s.io/?q=state%3A%20(dirty%7Cfree)&i=nope&files=&excludeFiles=&repos=kubernetes/k8s.io,kubernetes/test-infra